ee/tabby-webserver/src/service/mod.rs

```diff
@@ -48,6 +49,11 @@ impl ServerContext {
             client: Client::default(),
             completion: worker::WorkerGroup::default(),
             chat: worker::WorkerGroup::default(),
+            mail_service: Arc::new(
+                new_email_service(db_conn.clone())
```

Comment:
when `email_settings` got updated, will the mail_service itself being recreated?

Reply:
No, but its internal connection to the SMTP server will. See https://github.com/TabbyML/tabby/pull/1233/files#diff-a12677b41800287b462468d8f5f0e7a8b29d0fd8473af70a40ceff21f3209e54R72